src/main/java/com/sonyericsson/jenkins/plugins/bfa/ScanLogAction.java

```diff
@@ -64,6 +73,47 @@ public String getLogText() throws IOException {
         return FileUtils.readFileToString(new File(run.getRootDir(), FILE_NAME), StandardCharsets.UTF_8);
     }
 
+    /**
+     * The start time of the current scan.
+     * @return time in milliseconds {@link System#currentTimeMillis}
+     */
+    @Exported
+    public long getStartTime() {
+        return startTime;
+    }
+
+    /**
+     * The end time of the current scan.
+     * @return time in milliseconds {@link System#currentTimeMillis}
+     */
+    @Exported
+    public Long getEndTime() {
+        return endTime;
+    }
+
+    /**
+     * To call when the scan is finished.
+     */
+    public void finished() {
```

Comment:
I would maybe make this package-private if possible or if not, add `@Restricted(NoExternalUse.class)`.

Reply:
See https://github.com/jenkinsci/build-failure-analyzer-plugin/pull/175/commits/aa846cfd31d12d4a4743aab405af584504ac769c